images/Dockerfile.ubuntu

```diff
@@ -15,23 +17,26 @@ ARG MODEL=generic
 # the default value is empty, which means the HWE kernel WILL be installed
 # if you want to disable the HWE kernel, set HWE to "-non-hwe"
 ARG HWE=""
+ARG BASE_IMAGE=ubuntu:23.04
```

Comment:
wait... this is making all systems come from this? Isnt like it was before much better? Also isnt it bnumping the ubuntu version at the same time?

Reply:
yes, we should have no defaults. I don't remember why this default is there, maybe I screwed up in some rebase. I will remove.